scripts/plh-spreadsheet/process-content-spreadsheet.ts

```diff
@@ -85,25 +85,25 @@ export function getContentSheets(fileName: string, workbook: xlsx.WorkBook): { c
     const contentList: ContentIndexRow[] = xlsx.utils.sheet_to_json(workbook.Sheets[contentListSheetName]);
 
     const conversationSheets: ConversationExcelSheet[] = contentList
-        .filter((contentListItem) => contentListItem.Flow_Type === "Conversation")
-        .filter((contentListItem) => workbook.Sheets[contentListItem.Flow_Name])
-        .filter((contentListItem) => contentListItem.status.trim() !== "draft")
+        .filter((contentListItem) => contentListItem.flow_type === "conversation")
+        .filter((contentListItem) => workbook.Sheets[contentListItem.flow_name])
+        .filter((contentListItem) => contentListItem.status !== undefined && contentListItem.status.trim() === "released")
```

Comment:
Changed this round since `"draft"` is the default in the documentation, so should be considered draft if not provided.

Reply:
Sounds good!

Reply:
@chrismclarke functional change here